test/groovy/MavenExecuteTest.groovy

```diff
@@ -0,0 +1,41 @@
+import junit.framework.TestCase
+import org.junit.Before
+import org.junit.Test
+
+class MavenExecuteTest extends AbstractPiperUnitTest {
+
+    Map dockerParameters
+    List shellCalls
+
+    @Before
+    void setUp() {
+        super.setUp()
+
+        shellCalls = []
+        dockerParameters = [:]
+
+        helper.registerAllowedMethod("dockerExecute", [Map.class, Closure.class],
+            { parameters, closure ->
+                dockerParameters = parameters
+                closure()
+            })
+        helper.registerAllowedMethod('sh', [String], {s -> shellCalls.add(s)} )
+    }
+
+    @Test
+    void testExecuteBasicMavenCommand() throws Exception {
+        def script = loadScript("test/resources/pipelines/mavenExecuteTest/executeBasicMavenCommand.groovy",)
```

Comment:
Why is there a comma after the string argument? (This is seen in all `loadScript` calls.)
This is why the following lines are indented, right?